src/nvim/api/vim.c

```diff
@@ -151,6 +151,7 @@ Dictionary nvim_get_hl(Integer ns_id, Dict(get_highlight) *opts, Arena *arena, E
 ///                - cterm: cterm attribute map, like |highlight-args|. If not set,
 ///                         cterm attributes will match those from the attribute map
 ///                         documented above.
+///                - force: boolean if true force update the highlight group when it exist.
```

Comment:
`boolean` is redundant if we say "if true". also "when it exists" to be inglish grammar.

Reply:
get it.